app/js/components/LabResultEntry.jsx

```diff
@@ -228,21 +245,44 @@ export class LabResultEntry extends PureComponent {
             && (<span className="range-header-text">NORMAL RANGE</span>)
           }
           <span className="encounter-form-componnent">
-            <EncounterFormPage
-              defaultValues={[
-                {
-                  type: "obs",
-                  path: "test-order-number",
-                  concept: "393dec41-2fb5-428f-acfa-36ea85da6666",
-                  value: state.orderNumber,
-                },
-              ]}
-              backLink="/"
-              encounterType={encounterType}
-              formContent={observations}
-              patient={patientHeaderDetail}
-              formId="result-entry-form"
-            />
+            {hasEncounter
+              ? (
+                <EncounterFormPage
+                  encounter={encounter}
+                  defaultValues={[
+                    {
+                      type: "obs",
+                      path: "test-order-number",
+                      concept: "393dec41-2fb5-428f-acfa-36ea85da6666",
```

Comment:
If it's not too much stress, can you get this concept uuid, from the CONSTANTS reducer in react component. My bad, I forgot about moving it.or I could update it in my next PR.

Reply:
Also instead of repeating `defaultValues` twice, we could use save it to a variable and call it.
I.e 
```
const EncounterFormPageDefaultValue ={[
                    {
                      type: "obs",
                      path: "test-order-number",
                      concept: "393dec41-2fb5-428f-acfa-36ea85da6666",
                      value: state.orderNumber,
                    },
                  ]
```
defaultValues={EncounterFormPageDefaultValue}
It's cleaner that way

Reply:
+1 to use the constants reducer